rep-0147.rst

```diff
@@ -341,6 +343,33 @@ To send a series of goals a list of GlobalPositions can be used.
   nav_msgs/GlobalPosition[] waypoints
 
 
+Pose Reporting
+------------------
+
+An aerial vehicle can share its geographic pose via ``geographic_msgs/GeoPoseStamped`` [#geoposestamped]_.
+The position coordinates should follow WGS-84 [#wgs84]_.
+
+EGM2008 [#egm]_ can be assumed as the altitude datum unless otherwise stated.
+If and when EGM2020 is released, this REP can be updated.
+Take caution integrating GPS data and terrain data as they frequently use different datums.
+
+The orientation field follows REP 103's [#REP103]_ recommendation. The orientation is the transform from the 
+local tangent plane [#ltpc]_ to the body frame.
+
+A vehicle flying facing east will have a unit identity quaternion for its orientation.
+
+x,y,z=0, w=1
```

Comment:
If there is a more standard way to document the value for the quaternion (perhaps yaml), please supply a reference to the syntax I should use.